website/docusaurus.config.js

```diff
@@ -239,7 +240,8 @@ const announcementBar = {
     id: "bar", // Any value that will identify this message.
     // content: `<b>WIP:</b> Текущая версия методологии находится на стадии разработки и некоторые детали <i>могут измениться</i>`,
     // backgroundColor: '#e6a700', // As caution by docusaurus (defaults was `#fff`)
-    content: `Если вы используете методологию <a href="/versions">(v0 / v1 / v2)</a> по работе или в личных проектах, то <a href="https://github.com/feature-sliced/documentation/issues/131" target="_blank" rel="noreferrer noopener">расскажите нам!</a>`,
+    // FIXME: (i18n) translate by locale later (how to?)
+    content: `If you are using the methodology <a href="/versions">(v0 / v1 / v2)</a> for work or personal projects, <a href="https://github.com/feature-sliced/documentation/issues/131" target="_blank" rel="noreferrer noopener">tell, us!</a>`,
```

Comment:
Кажется, должно быть так 🤔
```suggestion
    content: `If you are using the methodology <a href="/versions">(v0 / v1 / v2)</a> at work or in personal projects, <a href="https://github.com/feature-sliced/documentation/issues/131" target="_blank" rel="noreferrer noopener">tell, us!</a>`,
```

Reply:
Машинный перевод + 3 часа ночи сделали свое дело так сказать 😄